### PR TITLE
22639 - Update Certify Section Wording

### PR DESCRIPTION
**Pull request by ArwenQin**
*Issue #:* /bcgov/entity#22639

*Description of changes:*
- This is to update the Filing UI Certify Section Wording
- Filing UI is not using the bcrs shared Certify component, updated the Certify.vue in common folder

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-filings-dev--pr-704-12r50d10.web.app

File: src/components/common/Certify.vue [hunk 1]
```diff
@@ -64,6 +74,9 @@ export default class Certify extends Vue {
 
   @Getter(useRootStore) getCurrentDate!: string
 
+  /** Is Staff prop. */
+  @Prop({ default: false }) readonly isStaff!: boolean
```
Comment (severinbeauvais) [inline src/components/common/Certify.vue:78]: Since this component already accesses the store, why not just add the getter here and simplify?

Comment (ArwenQin) [reply src/components/common/Certify.vue:78]: Got it, will update

**Review by severinbeauvais — APPROVED** [review]
LGTM

Sample:

image

File: src/components/common/Certify.vue (hunk 1 above)
Comment (ArwenQin) [reply src/components/common/Certify.vue]: Updated, is it ready to merge now?

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-filings-dev--pr-704-12r50d10.web.app

**Review by severinbeauvais — APPROVED** [review]
Looks good to me. Did you test it? If it's OK then go ahead and merge 😃

**Pull request merged**